### PR TITLE
Feature/noid/more remote testing

### PR DESCRIPTION
**Pull request by nickvergessen**

### CODE REVIEW

**Comment by nickvergessen** [comment]
Still not fully done, but another step taken and since it changes non-testing files I'd like to merge it to avoid future conflicts

File: tests/integration/run.sh [hunk 1]
```diff
@@ -18,7 +24,13 @@ ${ROOT_DIR}/occ app:enable spreed || exit 1
 ${ROOT_DIR}/occ app:enable spreedcheats || exit 1
 ${ROOT_DIR}/occ app:list | grep spreed
 
+# Disable bruteforce protection because the integration tests do trigger them
+${ROOT_DIR}/occ config:system:set auth.bruteforce.protection.enabled --value false --type bool
```
Comment (danxuliu) [inline tests/integration/run.sh:28]: Is this needed in Talk tests too?

**Review by danxuliu — COMMENTED** [review]
Code looks good :+1: 

But... I would like to remove the integration test from the second commit, as it does not work and I found it misleading when doing the review, and move the third commit to the end, as it depends on its following commits to work. Yes, you hate me, I know :-P

File: tests/integration/run.sh (hunk 1 above)
Comment (nickvergessen) [reply tests/integration/run.sh:28]: As soon there will be bruteforce protection being added to the endpoint, yes.

**Review by danxuliu — APPROVED** [review]
I would still split the fourth commit and squash the related changes with the third... but ok, I will settle with this :-P

**Comment by nickvergessen** [comment]
Drone conflict after #6730 

Rebased

**Review by gary-kim — APPROVED** [review]
Looks good to me!

**Pull request merged**